src/components/ScheduleModal.vue

```diff
@@ -138,4 +141,7 @@ async function saveChanges() {
     });
   return alert.present();
 }
+function goToCronHelp() {
+  window.open('https://www.freeformatter.com/cron-expression-generator-quartz.html',"_blank")
```

Comment:
instead of functions, why not use button href property?
https://ionicframework.com/docs/api/button#href